DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: crmPack
 Title: Object-Oriented Implementation of CRM Designs
-Version: 1.0.9000.9134
```

Comment:
yeah I would not bump this up manually, should be done automatically later by GH actions

Reply:
I posted in Slack about this earlier today.  I have already reverted by hand.